### PR TITLE
Alpha Release Candidate (Name TBD)

### PR DESCRIPTION
**Pull request by hlydecker**

### CODE REVIEW

File: data/filtered_nsw_sa1.geojson
Comment (sahandv) [inline data/filtered_nsw_sa1.geojson:1]: Is this file intended to stay?

File: .DS_Store
Comment (sahandv) [inline .DS_Store:1]: We can remove this one I guess. Right?

Comment (hlydecker) [reply .DS_Store:1]: Removed :)

File: data/filtered_nsw_sa1.geojson
Comment (hlydecker) [reply data/filtered_nsw_sa1.geojson:1]: Removed :)
